learners/setup.md

```diff
@@ -2,53 +2,154 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+The vantage6 has requirements on hardware and software to run its server and node instances. As for the detailed requirements, please refer to the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/requirements.html).
 
-## Data Sets
+For now, we just need to focus on the software setup.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+## Software setup
 
-## Software Setup
+This lesson requires the following software to be installed on your system:
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Operating system: Ubuntu 18.04+ , MacOS Big Sur+, or Windows 10+
```

Comment:
I would opt for Ubuntu 20.04+ as 18.04 is no longer in maintenance. ( we probably should also update that in the docs)
![image](https://github.com/vantage6/vantage6-workshop/assets/15981243/a0441ef9-c4d9-44a5-b0aa-99b2f5a6314b)

Also these OSes are recommended, So I would change it to: *Recommended Operating systems*

Reply:
@CunliangGeng I'm guessing you got the Ubuntu 18+ from our documentation? Then I guess we should also update there

Reply:
I just checked and indeed docs were outdated. Included an update in PR https://github.com/vantage6/vantage6/pull/1329

Reply:
Not only the docs but chapter 6 needs an [update](https://github.com/vantage6/vantage6-workshop/pull/29/commits/6ddd9dbcb51c8001cacc9a8a911cc4f47d286fef).

Reply:
It looks a bit annoying to update the ubuntu version from time to time. Shall we change the requirements to e.g.  "Ubuntu ([version with standard support](https://wiki.ubuntu.com/Releases))"?